### PR TITLE
nwhacks set up

### PR DESCRIPTION
**Pull request by vincentchiang1**
## Description

followed nick's hackcamp setup https://github.com/nwplus/external-templates/pull/135

### CODE REVIEW

File: .github/workflows/firebase_deploy.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ on:
         description: 'Targetted hackathon to build'
         required: true
   push:
-    branches: [cmd-f_dev, cmd-f_main, hackcamp2021_dev, hackcamp2021_main]
+    branches: [cmd-f_dev, cmd-f_main, hackcamp2021_dev, hackcamp2021_main, nwhacks2021_dev, nwhacks2021_main]
```
Comment (andykwan1124) [inline .github/workflows/firebase_deploy.yaml:10]: I think these should be nwhacks2022?

**Review by andykwan1124 — COMMENTED** [review]
Other than the action file, lgtm!

**Review by andykwan1124 — APPROVED** [review]
lesgooo

**Review by kevin-zou — COMMENTED** [review]
:shipit: good 🚢

**Pull request merged**